apps/shelter-web/src/app/shared/clients/apollo/client.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  ApolloClient,
+  from,
+  InMemoryCache,
+  NormalizedCacheObject,
+} from '@apollo/client';
+
+import { RestLink } from 'apollo-link-rest';
+import createUploadLink from 'apollo-upload-client/createUploadLink.mjs';
+import { csrfLink } from './csrf';
+
+type IApolloClient = {
+  apiUrl: string;
+};
+
+export const createApolloClient = (
+  props: IApolloClient
+): ApolloClient<NormalizedCacheObject> => {
+  const { apiUrl } = props;
+
+  const headers = { Referer: apiUrl };
```

Comment:
shouldn't need to be done since this was only required for mobile.  I don't think you can actually modify this in web.